index.bs

```diff
@@ -24,6 +24,68 @@ Assume Explicit For: yes
 Blah blah blah.
 </section>
 
+<h2 id="deployed-mitigations">
+Some browsers have deployed and announced protections against
+navigational-tracking.  This section is a work in progress to detail what
+protections have been shipped and / or are planned. This section should not
+be understood to be comprehensive.
+
+<h3>Safari</h3>
+Safari uses a heuristic-based approach to combat navigational-tracking. Safari uses
```

Comment:
We call it algorithmic since other browser vendors have tried to throw shade based on the term "heuristic."

Reply:
Happy to make this change, though will just point out that "is it in a list" is an algorithm too ;)